defold-spine/src/comp_spine_model.cpp

```diff
@@ -418,10 +418,18 @@ namespace dmSpine
         message.m_Playback    = track.m_Playback;
         message.m_Track       = entry->trackIndex + 1;
 
-        dmGameObject::Result result = dmGameObject::PostDDF(&message, &sender, &receiver, track.m_AnimationCallbackRef, true);
-        if (result != dmGameObject::RESULT_OK)
+        if (track.m_CallbackInfo)
         {
-            dmLogError("Could not send animation_done to listener: %d", result);
+            RunTrackCallback(track.m_CallbackInfo, dmGameSystemDDF::SpineAnimationDone::m_DDFDescriptor, (const char*)&message, &sender, true);
```

Comment:
not sure, maybe `RunTrackCallback` should return `dmGameObject::Result` as well?